api/envoy/config/accesslog/v3/accesslog.proto

```diff
@@ -87,6 +88,9 @@ message AccessLogFilter {
 
     // Metadata Filter
     MetadataFilter metadata_filter = 12;
+
+    // Access Log Type Filter
+    AccessLogTypeFilter access_log_type_filter = 13;
```

Comment:
nit: would simple `log_type_filter` or `type_filter` be better? because all these filter are applied to access log. seems the `access_log` prefix is unnecessary?

Reply:
I agree, done